.github/workflows/sonarqube.yml

```
@@ -12,16 +12,97 @@ jobs:
+          pip install pytest pytest-django pytest-cov
+
+      - name: Run migrations
+        working-directory: ./backend
+        run: python manage.py migrate
+
+      # Fix the coverage generation with modified path
+      - name: Run tests with coverage
+        working-directory: ./backend
+        run: |
+          pytest
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)